sensu/transport/rabbitmq/rabbitmq.go

```diff
@@ -2,30 +2,86 @@ package rabbitmq
 
 import (
 	"errors"
-	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/upfluence/goutils/log"
+	"fmt"
+	"math/rand"
+	"time"
 
-	"github.com/upfluence/sensu-go/Godeps/_workspace/src/github.com/streadway/amqp"
+	"github.com/upfluence/sensu-client-go/Godeps/_workspace/src/github.com/streadway/amqp"
```

Comment:
the path reference the dependencies vendored in `github.com/upfluence/sensu-client-go`, it should be referencing the one in `github.com/upfluence/sensu-go`